# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1497839121 14400
#      Sun Jun 18 22:25:21 2017 -0400
# Node ID 037d65ae2b601781325d30189b859c71373bf8bb
# Parent  b488e958d024183138e960cbb965f4305e5a2cbc
Backed out changeset fca2f775ab4c

diff --git a/libinterp/parse-tree/oct-parse.in.yy b/libinterp/parse-tree/oct-parse.in.yy
--- a/libinterp/parse-tree/oct-parse.in.yy
+++ b/libinterp/parse-tree/oct-parse.in.yy
@@ -1448,24 +1448,17 @@ file            : INPUT_FILE opt_nl opt_
                       }
 
                     $$ = 0;
                   }
                 | INPUT_FILE opt_nl classdef parsing_local_fcns opt_sep opt_fcn_list END_OF_INPUT
                   {
                     YYUSE ($2);
                     YYUSE ($5);
-
-                    // Delete the dummy statement_list we created
-                    // after parsing the function.  Any local function
-                    // definitions found in the file have already been
-                    // stored in the symbol table or in
-                    // base_parser::primary_fcn_ptr.
-
-                    delete $6;
+                    YYUSE ($6);
 
                     if (lexer.reading_classdef_file)
                       parser.classdef_object = $3;
 
                     $$ = 0;
                   }
                 ;
 
